modules/ROOT/pages/apikit-graphql-module-reference.adoc

```diff
@@ -0,0 +1,244 @@
+* <<RepeatableFileStoreStream>>
+* non-repeatable-stream |  Configure if repeatable streams should be used and their behavior. |  | 
+| Target Variable | String |  The name of a variable on which the operation output is placed. |  | 
+| Target Value | String |  An expression that is evaluated against the operation output and the outcome of that expression is stored in the target variable. |  #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Any
+|===
+
+==== For Configurations
+
+* <<Config>> 
+
+=== Serialize
+
+`<graphql-router:serialize>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration you use. | | *x*
+| Inbound Content | Any |  |  #[payload] | 
+| Config Ref | ConfigurationProvider |  The name of the configuration you use to execute this component. |  | *x*
+| Object Type Name | String |  The object type name |  | *x*
+| Field Name | String |  The field name |  | 
+| Target Variable | String |  The name of a variable on which the operation output is placed. |  | 
+| Target Value | String |  An expression that is evaluated against the operation output and the outcome of that expression is stored in the target variable. |  #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Any
+|===
+
+==== For Configurations
+
+* <<Config>> 
+
+=== Serialize Batch
+
+`<graphql-router:serialize-batch>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration you use. | | *x*
+| Object Type Name | String |  |  | *x*
+| Inbound Content | Any |  |  #[payload] | 
+| Config Ref | ConfigurationProvider |  The name of the configuration you use to execute this component. |  | *x*
+| Target Variable a| String |  The name of a variable on which the operation output is placed. |  | 
+| Target Value | String |  An expression that is evaluated against the operation output and the outcome of that expression is stored in the target variable. |  #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Any
+|===
+
+==== For Configurations
+
+* <<Config>> 
+
+== Sources
+
+=== Data Fetcher
+
+`<graphql-router:data-fetcher>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration you use. | | *x*
+| Output MIME Type a| String |  The MIME type of the payload that this operation outputs. |  | 
+| Output Encoding a| String |  The encoding of the payload that this operation outputs. |  | 
+| Config Ref | ConfigurationProvider |  The name of the configuration you use to execute this component. |  | *x*
+| Primary Node Only a| Boolean |  Whether this source should only be executed on the primary node when running in cluster. |  | 
+| Streaming Strategy a| 
+* <<RepeatableInMemoryStream>>
+* <<RepeatableFileStoreStream>>
+* non-repeatable-stream |  Configure if repeatable streams should be used and their behavior. |  | 
```

Comment:
Should it link to "=== Repeatable File Store Stream" title?
If so, we link to content with an xref: xref:filename.adoc#articletitle[title or phrase to explain title action]

Reply:
OK! Thanks!